lib/drivers/animasDriver.js

```diff
@@ -1759,7 +1760,7 @@ module.exports = function (config) {
         postrecords.push(bgRecord);
       }
       else {
-        console.log('Discarding control solution test');
+        debug('Discarding control solution test');
```

Comment:
I'm only including the `debug` statements that are very chatty in the `verbose` clause. The ones that are not wrapped don't appear that often, and could provide valuable debugging info when on prod.